libtransmission/file-win32.c

```diff
@@ -733,6 +733,23 @@ bool tr_sys_path_rename(char const* src_path, char const* dst_path, tr_error** e
     return ret;
 }
 
+bool tr_sys_path_copy(char const* src_path, char const* dst_path, uint64_t unused UNUSED, tr_error** error)
+{
+    DWORD const flags =
+#ifdef COPY_FILE_ALLOW_DECRYPTED_DESTINATION
+        COPY_FILE_ALLOW_DECRYPTED_DESTINATION;
+#else
+        0;
+#endif
+    if (CopyFileExA(src_path, dst_path, NULL, NULL, false, flags) == 0)
```

Comment:
Only Unicode ("W") APIs are allowed for file operations on Windows. ANSI ("A") APIs expect the path(s) in ANSI/OEM encoding, and the paths inside libtransmission are always in UTF-8 (or at least they try to be). Another shortcoming of ANSI APIs is that they limit the valid path length to `MAX_PATH` (260 characters) while Unicode APIs allow for the limit to be extended to 32 KiB. See https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file for more information.

Next to last argument to the function has `LPBOOL` type which is neither `bool` (`false` that you're passing) nor `BOOL` (`FALSE` that you probably wanted to pass). You're effectively passing `NULL` here but I don't see any indication in the documentation of this parameter being optional, so I'd feel safer if you define a `BOOL` variable and pass its address instead.